packages/react/src/components/Nav/Nav.types.ts

```diff
@@ -168,6 +168,10 @@ export interface INavLinkGroup {
    * (Optional) Any additional properties to apply to a group.
    */
   groupData?: any;
+  /**
+   * Provides consumer control to manually update the collapsed state of the Nav.
+   */
+  isCollapsed?: boolean;
```

Comment:
the `INavLink` prop is `isExpanded`, so it might make sense to have the `INavLinkGroup` prop also be `isExpanded` to avoid confusion especially since the two interfaces are already pretty similar.